tests/conftest.py

```diff
@@ -111,23 +111,40 @@ def respond_invitation(selenium, request_page, url, accept, quota=None, comment=
         container = selenium.find_element(By.CLASS_NAME, 'note_editor')
 
         buttons = container.find_elements(By.TAG_NAME, "button")
-        assert len(buttons) == 2
+        
+        if accept and quota:
```

Comment:
I do think you need to check the length of the buttons in order to know if the invitation allows accept with reduce quota. 

Previous tests send accept=true and quota=true and that means, we need to decline first and then accept with reduce quota.

The logic would be:

```
if accept and quota:

     if len(buttons) == 3:
         buttons[1].click() ## accept with reduced quota, new implemented button that is shown in the middle
         ... select quota, etc

     if len(buttons) == 2:
         buttons[1].click(). ## decline first
         ...continue with the original code....


```

Reply:
Yeah I saw the previous tests pass `accept=true` even though they're actually declining with reduced load. What do you think about changing them to `accept=false`? I feel it would make sense since they're declining with quota, but it would mean changing previous tests.